api/v1alpha1/openapi.yaml

```diff
@@ -713,6 +713,44 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /api/v1/devices/{name}/rendered:
```

Comment:
Why do we move it to a separate endpoint. Would it make sense to make it part of the regular device get?

Reply:
The regular device get is for users/UI.  This new endpoint is for the agent.

Reply:
I would just name it /api/v1/devices/{name}/config